openlibrary/i18n/te/messages.po

```diff
@@ -275,78 +275,78 @@ msgstr "మీ ప్రస్తుత సంకేతపదం"
 
 #: borrow.html:85
 msgid "Loan Expires"
-msgstr ""
+msgstr "రుణం గడువు"
 
 #: borrow.html:86
 #, fuzzy
 msgid "Loan actions"
-msgstr "చర్యలు"
+msgstr " రుణం చర్యలు"
 
 #: borrow.html:114
 #, python-format
 msgid "There is one person waiting for this book."
 msgid_plural "There are %(n)s people waiting for this book."
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "ఒకేఒక్క మనిషి మాత్రం ఈ పుస్తకం కోసం వేచిఉన్నాడు."
+msgstr[1] "అక్కడ %n (s ) మనుషులు ఈ పుస్తకం కోసం వేచిఉన్నారు."
 
 #: borrow.html:151
 msgid "Books You're Waiting For"
-msgstr ""
+msgstr "మీరు ఎదురు చూస్తున్న పుస్తకాలు "
 
 #: borrow.html:190
 #, fuzzy, python-format
```

Comment:
for this `fuzzy`, the fuzzy should be removed, but `python-format` kept
so the line becomes:
`#, python-format`

Reply:
Should this be మీరు వేచి ఉన్న పుస్తకాలు?

Reply:
“మీరు ఎదురు చూస్తున్న పుస్తకాలు” is correct.